### ISSUE TITLE
Update this extension to run with latest sysl version

### ISSUE THREAD

**Issue by joshcarp** [anz-bank/vscode-sysl#16]
Vscode extension currently doesn't support latest updated grammar (for syntax highlighting, at least)

Language server should also be able to compile sysl without needing sysl installed (nice to have)
Maybe looking into running wasm; I've had some experience with this but it'd be nice to have someone to help with this.

**Comment by nydrani** [comment]
I dont think it's a good idea to compile sysl without having sysl installed since that would mean both the extension would have to package a static binary which would result in a possible divergence in compilation results compared to installed system sysl.

It sounds a bit ridiculous to me, if someone were to have the vscode-sysl lsp extension installed and not sysl. Most other lsp implementations ask you to install the compiler for the specific language if it is not found on the system.

Updated syntax highlighting according to @nofun97 may be blocked by the wbnf grammar update.
